products/oracle-database.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Oracle Database
+category: db
+iconSlug: oracle
+permalink: /oracle-database
+versionCommand: show version
+releasePolicyLink: https://www.oracle.com/us/assets/lifetime-support-technology-069183.pdf
```

Comment:
HTML links should be favored, such as https://www.oracle.com/support/lifetime-support/software.html.

Reply:
> HTML links should be favored, such as https://www.oracle.com/support/lifetime-support/software.html.

:bulb: Could this lint be part of the CI :grey_question: